editor/src/templates/image-dnd.spec.browser2.tsx

```diff
@@ -240,7 +241,7 @@ describe('image dnd', () => {
     sandbox.restore()
   })
 
-  it('dragging from the sidebar works', async () => {
+  xit('dragging from the sidebar works', async () => {
```

Comment:
Should this test still be disabled?

Reply:
it's disabled on as long as https://github.com/concrete-utopia/utopia/pull/2659 isn't merged, so once it is, I'm aiming to fix it in a followup PR